client/src/components/Applications/AppBrowser/AppBrowser.scss

```diff
@@ -71,11 +73,6 @@ ul#appBrowser-sidebar {
   flex-grow: 1;
 
   background-color: #f4f4f4;
-  border-left: 1px solid rgba(112, 112, 112, 0.25);
-
-  .apps-grid-item {
-    min-width: var(--min-width);
-  }
```

Comment:
__Explanation__
The location of these styles are moved (to [above](https://github.com/TACC/Core-Portal/pull/350/files#diff-c34da383f5822d39f8f3f31609ffb6aacfe3254f371dd63125a47c8ddde98755R49) and [more above](https://github.com/TACC/Core-Portal/pull/350/files#diff-c34da383f5822d39f8f3f31609ffb6aacfe3254f371dd63125a47c8ddde98755R37)) to make app sidebar styles organized similar to standard sidebars.